func-complete-check/check-validator/v1.ts

```diff
@@ -0,0 +1,81 @@
+import { Context } from '@azure/functions'
+import { ValidateCheckMessageV1, ReceivedCheck, MarkCheckMessageV1 } from '../typings/message-schemas'
+import Moment from 'moment'
+import * as R from 'ramda'
+import compressionService from '../lib/compression-service'
+import azureStorageHelper from '../lib/azure-storage-helper'
+const tableService = azureStorageHelper.getPromisifiedAzureTableService()
+import checkSchema from '../messages/complete-check.v1.json'
+
+class V1 {
+  async process (context: Context, validateCheckMessage: ValidateCheckMessageV1) {
+    let receivedCheck = findReceivedCheck(context.bindings.receivedCheckTable)
+    try {
+      detectArchive(receivedCheck)
+      const decompressedString = compressionService.decompress(receivedCheck.archive)
+      const checkData = JSON.parse(decompressedString)
```

Comment:
do we need to safeguard the parsing at this level?
just curious about the output of the lzstring decompress.

Reply:
yes definitely, good point.  this code is not production ready, but thats an important part of it

Reply:
i will add this to the ACs of the PBI